ledger-tool/src/ledger_utils.rs

```diff
@@ -305,6 +307,25 @@ pub fn load_and_process_ledger(
         "Using: block-verification-method: {}",
         block_verification_method,
     );
+    match block_verification_method {
+        BlockVerificationMethod::BlockstoreProcessor => {
+            info!("no scheduler pool is installed for block verification...");
+        }
+        BlockVerificationMethod::UnifiedScheduler => {
+            let no_transaction_status_sender = None;
```

Comment:
this particular line introduced this bug...: https://github.com/anza-xyz/agave/pull/3861